packages/liveblocks-core/src/room.ts

```diff
@@ -1016,7 +1016,9 @@ function makeStateMachine<
       if (context.socket === null) {
         throw new Error("Can't send message if socket is null");
       }
-      context.socket.send(JSON.stringify(messageOrMessages));
+      if (context.socket.readyState === context.socket.OPEN) {
+        context.socket.send(JSON.stringify(messageOrMessages));
```

Comment:
should we log or queue this message? I understand in the current code this might throw, but maybe we should also handle that? I'm afraid this might mask an undesirable behavior rather than solving it.

Reply:
Yes, I think we do want to do that. I didn't want to go on a tangent there for this PR, but this is something I think we need to do better as part of the stability improvement. Still not sure what the final result for that will look like, but I'll put this on my todo list.